.depcheckrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ignores": [
+    "@lavamoat/allow-scripts",
+    "@lavamoat/preinstall-always-fail",
```

Comment:
This is referencing tools we aren't using here yet. Maybe this can wait until after the allow scripts PR is merged

Reply:
Yup, good call. That PR is merged now.